tools/docker-compose/ansible/roles/sources/templates/haproxy.cfg.j2

```diff
@@ -32,15 +32,15 @@ backend nodes
     option httpchk HEAD / HTTP/1.1\r\nHost:localhost
 {% for i in range(control_plane_node_count|int) %}
     {% set container_postfix = loop.index %}
-    server tools_awx_{{ container_postfix }} tools_awx_{{ container_postfix }}:8013 check
+    server tools_awx_{{ container_postfix }} tools_awx_{{ container_postfix }}:8013 check inter 10s
```

Comment:
can someone help me understand why this change was made?

Reply:
per a convo @relrod we understand what it does, changing how often we check the health of the backend servers. Why this change was made, we may need to poke @shanemcd on that as he made this change